<?xml version="1.0" encoding="UTF-8"?><ui version="4.0"> <class>MainWindow</class> <widget class="QMainWindow" name="MainWindow">  <property name="geometry">   <rect>    <x>0</x>    <y>0</y>    <width>671</width>    <height>629</height>   </rect>  </property>  <property name="minimumSize">   <size>    <width>600</width>    <height>0</height>   </size>  </property>  <property name="maximumSize">   <size>    <width>800</width>    <height>16777215</height>   </size>  </property>  <property name="windowTitle">   <string>RtShowYou</string>  </property>  <widget class="QWidget" name="centralwidget">   <layout class="QVBoxLayout" name="verticalLayout">    <item>     <widget class="QWidget" name="widget_2" native="true">      <property name="minimumSize">       <size>        <width>0</width>        <height>56</height>       </size>      </property>      <property name="maximumSize">       <size>        <width>16777215</width>        <height>56</height>       </size>      </property>      <layout class="QHBoxLayout" name="horizontalLayout_2">       <property name="spacing">        <number>10</number>       </property>       <property name="leftMargin">        <number>0</number>       </property>       <property name="topMargin">        <number>10</number>       </property>       <property name="rightMargin">        <number>0</number>       </property>       <property name="bottomMargin">        <number>10</number>       </property>       <item>        <widget class="QWidget" name="widget" native="true">         <layout class="QHBoxLayout" name="horizontalLayout">          <property name="leftMargin">           <number>0</number>          </property>          <property name="topMargin">           <number>0</number>          </property>          <property name="rightMargin">           <number>0</number>          </property>          <property name="bottomMargin">           <number>0</number>          </property>          <item>           <widget class="QPushButton" name="pushButton">            <property name="sizePolicy">             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">              <horstretch>0</horstretch>              <verstretch>0</verstretch>             </sizepolicy>            </property>            <property name="minimumSize">             <size>              <width>100</width>              <height>0</height>             </size>            </property>            <property name="maximumSize">             <size>              <width>100</width>              <height>16777215</height>             </size>            </property>            <property name="text">             <string>Key word:</string>            </property>            <property name="flat">             <bool>true</bool>            </property>           </widget>          </item>          <item>           <widget class="QLineEdit" name="lineEdit_send">            <property name="sizePolicy">             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">              <horstretch>0</horstretch>              <verstretch>0</verstretch>             </sizepolicy>            </property>            <property name="styleSheet">             <string notr="true">padding-left: 2px;</string>            </property>            <property name="text">             <string>ShowReeToo</string>            </property>            <property name="clearButtonEnabled">             <bool>false</bool>            </property>           </widget>          </item>         </layout>        </widget>       </item>       <item>        <widget class="QPushButton" name="pushButton_send">         <property name="sizePolicy">          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">           <horstretch>0</horstretch>           <verstretch>0</verstretch>          </sizepolicy>         </property>         <property name="minimumSize">          <size>           <width>120</width>           <height>0</height>          </size>         </property>         <property name="text">          <string>Search</string>         </property>        </widget>       </item>      </layout>     </widget>    </item>    <item>     <widget class="QTableView" name="tableView">      <property name="styleSheet">       <string notr="true">QHeaderView::section:first{    color:black;    height: 40px;} QHeaderView::section:middle{    color:black;    height: 40px;} QHeaderView::section:last{    color:black;    height: 40px;}</string>      </property>      <property name="editTriggers">       <set>QAbstractItemView::DoubleClicked</set>      </property>      <property name="alternatingRowColors">       <bool>true</bool>      </property>      <property name="selectionMode">       <enum>QAbstractItemView::NoSelection</enum>      </property>      <property name="sortingEnabled">       <bool>true</bool>      </property>      <attribute name="horizontalHeaderMinimumSectionSize">       <number>100</number>      </attribute>      <attribute name="horizontalHeaderDefaultSectionSize">       <number>200</number>      </attribute>      <attribute name="horizontalHeaderStretchLastSection">       <bool>true</bool>      </attribute>      <attribute name="verticalHeaderVisible">       <bool>false</bool>      </attribute>      <attribute name="verticalHeaderMinimumSectionSize">       <number>30</number>      </attribute>      <attribute name="verticalHeaderDefaultSectionSize">       <number>35</number>      </attribute>     </widget>    </item>    <item>     <widget class="QPlainTextEdit" name="plainTextEdit">      <property name="sizePolicy">       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">        <horstretch>0</horstretch>        <verstretch>0</verstretch>       </sizepolicy>      </property>      <property name="minimumSize">       <size>        <width>0</width>        <height>150</height>       </size>      </property>      <property name="maximumSize">       <size>        <width>16777215</width>        <height>150</height>       </size>      </property>      <property name="verticalScrollBarPolicy">       <enum>Qt::ScrollBarAsNeeded</enum>      </property>      <property name="horizontalScrollBarPolicy">       <enum>Qt::ScrollBarAsNeeded</enum>      </property>      <property name="plainText">       <string/>      </property>     </widget>    </item>   </layout>  </widget>  <widget class="QStatusBar" name="statusBar"/> </widget> <resources/> <connections/></ui>